airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/oauth.py

```diff
@@ -104,9 +105,29 @@ def __init__(
         refresh_token_name: str = "refresh_token",
         refresh_request_body: Mapping[str, Any] = None,
         grant_type: str = "refresh_token",
-        credentials_configuration_field_name: str = "credentials",
+        client_id_config_path="/credentials/client_id",
```

Comment:
I would recommend to make this a list instead to avoid problems with special characters (dpath supports it like `["credentials", "client_secret"]`

Reply:
👍 I defaulted to tuples and declared these parameters as `Sequence` type that dpath.get can handle according to what I understood from their implementation.